i18n/locales/en.ts

```diff
@@ -1210,7 +1210,7 @@ const translations = {
   },
 
   chapter_six: {
-    title: `Offense is the best defense`,
+    title: `The Keyholder`,
```

Comment:
Just an FYI that on prod, chapter 5 is also called "The Keyholder"

Reply:
doh!